developing-with-nats/tutorials/jwt.md

```diff
@@ -956,17 +955,17 @@ In case you generate a user on behalf of another entity that has no nsc environm
 ### Automated sign up services - JWT and NKEY libraries
 
 `nsc` essentially uses the [NKEY](https://github.com/nats-io/nkeys) and [JWT](https://github.com/nats-io/jwt) libraries to generate operator/accounts/users.
-You can use these libraries to generate the necessary artifacts as too.
+You can use these libraries to generate the necessary artifacts as well.
 Generating the operator makes little sense, Accounts only if you need them dynamically, say for everyone of your customer. 
```

Comment:
'Generating the operator makes little sense, Accounts only if you need them dynamically, say for everyone of your customer. ' I'm not sure I understand this sentence. May need to reword?

Reply:
changed to:
Because there is only one, generating the operator this way makes little sense. 
Accounts only if you need them dynamically, say for everyone of your customer.